config/components/rbac/role.yaml

```diff
@@ -1,163 +1,180 @@
+
```

Comment:
nitpick: spacing

Reply:
This space is added whenever the file is regenerated :/